src/translate/translate.py

```diff
@@ -753,3 +754,6 @@ def handle_sigxcpu(signum, stackframe):
         traceback.print_exc(file=sys.stdout)
         print("=" * 79)
         sys.exit(TRANSLATE_OUT_OF_MEMORY)
+    except pddl_parser.ParseError as e:
+        print(e)
+        sys.exit(TRANSLATE_INPUT_ERROR)
```

Comment:
This changes the meaning of `TRANSLATE_INPUT_ERROR` compared to what we say on the wiki ("Usage error: wrong command line options"). If nobody things we need two different error codes for command-line errors vs. input file errors, the code can stay is is, but then we should update the wiki documentation to cover both cases.